Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex

```diff
@@ -118,11 +118,15 @@ \subsubsection{REST API}
 
 \subsubsection{Music matching algorithm}\label{MatchingAlgorithm}
 The music matching algorithm determines and compares moods for songs, rooms and users.
-Moodcat will be developed around a learning algorithm.
-This means that Moodcat will improve over time.
-Default values will be determined by examining the audio features from our data set.
-Then, through a system of user votes, the system will learn and improve over time.
+The classification of songs will be done by classification from the user, which will be encouraged with gamification.
+Moodcat will improve over time, because the more people classify the more accurate it gets.
+By default, a song will be classified with the default vector: valence 0 and arousal 0. This would mean an absolute neutral mood. Over time the values will move more to the actual values. These lie between -1 and 1.
```

Comment:
'All entities are modeled with a valence and arousal vector. Valence and arousal are measured in a range of -1 to 1. This classification (mapping) will be updated over time by processing user feedback. Therefore the longer the system is running, the more accurate the classifications become.'